libraries/mbed/hal/can_api.h

```diff
@@ -37,7 +37,8 @@ typedef enum {
     IRQ_PASSIVE,
     IRQ_ARB,
     IRQ_BUS,
-    IRQ_READY
+    IRQ_READY,
+    IRQ_RX1
```

Comment:
Hello @0xc0170,
I can remove RX1 interrupt. It will take longer to receive 2 messages (no more parallel reception). I find it a pity not to use the HW capabilities, but we can do it if you like.
I don't understand the impact of adding an a value in the enum type on other platforms. They all have a check on the IRQ value, and don't handle it when they don't recognize the irq id.
Let me know your decision, I'll implement it.
Cheers